### PR TITLE
Update /firefox/mobile/ page CTA buttons (Fixes #8250)

### LINKED ISSUE #8250: mobile page CTA update - button
## Description

We need to remove the copy CTA "Get Firefox Mobile" and phone image, replacing that copy in a big blue button and putting that button above the badges - see the attached image for reference.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Changes primary and secondary CTA buttons on `/firefox/mobile/` to be standard product themed buttons.

## Issue / Bugzilla link
#8250

## Testing
- [x] Both primary and secondary modals should continue to work the same.

Successful test run: https://gitlab.com/mozmeao/www-config/pipelines/102037189

### CODE REVIEW

File: bedrock/firefox/templates/firefox/mobile-2019.html [hunk 1]
```diff
@@ -41,6 +41,11 @@ <h3>{{ _('Get automatic privacy on mobile') }}</h3>
           </div>
 
           <div class="mzp-c-hero-cta">
+            <div class="header-product-ctas">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/mobile-2019.html]: I think this needs `hide-from-legacy-ie` on it? Until we can make it a link to the get-app page.

**Comment by alexgibson** [comment]
Thanks @stephaniehobson updated

**Comment by stephaniehobson** [comment]
Needs a rebase because of #8291 otherwise looking good.

**Comment by alexgibson** [comment]
Thanks @stephaniehobson updated

**Pull request merged**